### PR TITLE
Text fixes

### PR DESCRIPTION
**Pull request by divine-comedian**
Various text changes, grammar fixes and capitalizations.

### CODE REVIEW

File: src/models/Donation.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ class Donation extends Model {
   }
 
   static get TO_APPROVE() {
-    return 'ToApprove';
+    return 'To Approve';
```
Comment (aminlatifi) [inline src/models/Donation.js:21]: No, you should not change it. It's mapped to status in backend.

**Review by aminlatifi — CHANGES_REQUESTED** [review]

File: src/models/Donation.js (hunk 1 above)
Comment (divine-comedian) [reply src/models/Donation.js]: fixed in commit below :point_down:

File: src/models/Donation.js [hunk 2]
```diff
@@ -22,32 +22,32 @@ class Donation extends Model {
   }
 
   static get WAITING() {
-    return 'Waiting';
```
Comment (aminlatifi) [inline src/models/Donation.js:25]: These ones also should be changed.

**Review by aminlatifi — CHANGES_REQUESTED** [review]

**Comment by aminlatifi** [comment]
Thanks @divine-comedian

**Pull request merged**